src/SFA.DAS.EmployerIncentives.Commands/EarningsResilienceCheck/EarningsResilienceCheckHandler.cs

```diff
@@ -1,48 +1,49 @@
 ﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
-using SFA.DAS.EmployerIncentives.Abstractions.Events;
-using SFA.DAS.EmployerIncentives.Data.EarningsResilienceCheck;
-using SFA.DAS.EmployerIncentives.Domain.EarningsResilienceCheck.Events;
-using SFA.DAS.EmployerIncentives.Domain.IncentiveApplications.Models;
+using SFA.DAS.EmployerIncentives.Commands.Exceptions;
+using SFA.DAS.EmployerIncentives.Commands.Persistence;
 using SFA.DAS.EmployerIncentives.Queries.EarningsResilienceCheck;
-using System.Collections.ObjectModel;
-using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.EmployerIncentives.Commands.EarningsResilienceCheck
 {
     public class EarningsResilienceCheckHandler : ICommandHandler<EarningsResilienceCheckCommand>
     {
-        private readonly IEarningsResilienceCheckRepository _resilienceCheckRepository;
-        private readonly IDomainEventDispatcher _domainEventDispatcher;
+        private readonly IIncentiveApplicationDomainRepository _applicationDomainRepository;
+        private readonly IApprenticeshipIncentiveDomainRepository _incentiveDomainRepository;
 
-        public EarningsResilienceCheckHandler(IEarningsResilienceCheckRepository resilienceCheckRepository, IDomainEventDispatcher domainEventDispatcher)
+        public EarningsResilienceCheckHandler(IIncentiveApplicationDomainRepository applicationDomainRepository,
+                                              IApprenticeshipIncentiveDomainRepository incentiveDomainRepository)
         {
-            _resilienceCheckRepository = resilienceCheckRepository;
-            _domainEventDispatcher = domainEventDispatcher;
+            _applicationDomainRepository = applicationDomainRepository;
```

Comment:
sorry i think this is gonna have to be split into 2 commands (1 for each bounded context) or a domain event in one & command handler in the other.